.ansible-lint

```diff
@@ -2,6 +2,8 @@
 exclude_paths:
     - ci/
     - roles/telemetry_autoscaling
+    - roles/telemetry_logging
```

Comment:
I'm not sure exactly but if I remember correctly, We had to add telemetry_autoscaling here as the role was running some commands from external repos and that was causing some problem , but I don't think we should continue do that for all the roles as that can cause alot of syntax problems later.  but we can have that later as a to-do task.

Reply:
Some of these should be addressed sooner rather than later. The role isn't loadable at the moment because of a syntax error in meta/main.yml

I'm using ansible lint locally to do some basic checking, and  I'll update this PR shortly